examples/flipper/Cargo.toml

```
@@ -1,16 +1,13 @@
 [package]
 name = "flipper"
-authors = ["Cardinal", "Aleph Zero Foundation"]
+authors = ["Use Ink <ink@use.ink>"]
```

```suggestion
authors = ["Cardinal", "Aleph Zero Foundation", "Use Ink <ink@use.ink>"]
```
Ditto about authors.